src/main/java/com/google/sps/data/DBReferences.java

```diff
@@ -0,0 +1,10 @@
+package com.google.sps.data;
```

Comment:
All capstone code should in your own named package.

Reply:
What is the proper way to name our package? Something like com.google.meltingpot.data?

Reply:
Like com.google.sps.meltingpot.data would be good.